### PR TITLE
Portal-Add org profile page and contact management page

### PR DESCRIPTION
**Pull request by linkfang**
Implemented the layout of these 2 pages. 
Preview here:
[Org Profile](https://deploy-preview-232--eclipsefdn-react-members.netlify.app/portal/org-profile)
[Contact Manangement](https://deploy-preview-232--eclipsefdn-react-members.netlify.app/portal/contact-management)

For [issue157](https://github.com/EclipseFdn/react-eclipsefdn-members/issues/157) 
For [issue158](https://github.com/EclipseFdn/react-eclipsefdn-members/issues/158)
Fixes [issue323](https://github.com/EclipseFdn/react-eclipsefdn-members/issues/323)
Fixes [issue324](https://github.com/EclipseFdn/react-eclipsefdn-members/issues/324)

### CODE REVIEW

File: src/main/www/src/components/Portal/ContactManagement/ContactManagement.tsx [hunk 1]
```diff
@@ -1,62 +1,329 @@
...
-      >
-        <h5>Table</h5>
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setRemoveReason((event.target as HTMLInputElement).value);
+  };
+
+  const handleRequestRemoval = () => {
+    setOpenRequestRemoval(false);
+    const newRows = contactList.filter((contact) => contact.id !== selectedContact.id);
+    console.log('Deleted ', selectedContact.name, '| Because of: ', removeReason, '| With comment: ', removeComment);
```
Comment (autumnfound) [inline src/main/www/src/components/Portal/ContactManagement/ContactManagement.tsx:209]: As we've merged in the console prod check, you should rebase and add it here

File: src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx [hunk 2]
```diff
@@ -1,9 +1,327 @@
...
+      </Link>
+    </div>
+  ));
+
   return (
     <>
-      <Typography variant="h4">Your Organizatino Profile</Typography>
+      <BusinessIcon className={classes.headerIcon} />
+      <Typography variant="h4" className={classes.pageHeader}>
+        Your Organizatino Profile
```
Comment (autumnfound) [inline src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx:127]: Typo in label

File: src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx [hunk 3]
```diff
@@ -1,9 +1,327 @@
...
+
+          <Grid item>
+            <Button className={classes.uploadBtn} onClick={() => setOpenLogoForWeb(true)}>
+              Upload New
+            </Button>
+          </Grid>
+
+          <Grid item>
+            <Typography className={classes.helperText} variant="body2">
+              The supported formats for uploading your logo include: PNG, JPG, or GIF file under 1 MB in size.
```
Comment (autumnfound) [inline src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx:174]: Chris had mentioned removing GIF support as animated images aren't supported atm. We may need to check on this desc

File: src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx [hunk 4]
```diff
@@ -1,9 +1,327 @@
...
+              onChange={formikOrg.handleChange}
+              error={formikOrg.touched.orgProfile?.companyURL && Boolean(formikOrg.errors.orgProfile?.companyURL)}
+              helperText={formikOrg.errors.orgProfile?.companyURL}
+            />
+          </Grid>
+        </Grid>
+
+        <Grid container spacing={2} className={classes.uploadCtn}>
+          <Grid item>
+            <Typography className={classes.subHeader} variant="h6">
```
Comment (autumnfound) [inline src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx:161]: This is out of order and should follow the other headings

**Comment by autumnfound** [comment]
Additionally, you should use an accessibility tool to make sure each of the form inputs have labels (some are missing labels atm)

File: src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx (hunk 4 above)
Comment (linkfang) [reply src/main/www/src/components/Portal/OrgProfile/OrgProfiles.tsx:161]: Do you mean to use variant="h5" here?

**Comment by linkfang** [comment]
> Additionally, you should use an accessibility tool to make sure each of the form inputs have labels (some are missing labels atm)

For some reason, Material-UI doesn't associate the label with the input element in a common/normal way. 
So, I added aria-label prop to the input component manually to fix the issue.

File: src/main/www/src/components/Portal/Dashboard/DashboardCommittersAndContributors.tsx [hunk 5]
```diff
@@ -120,20 +120,20 @@ export default function DashboardCommittersAndContributors() {
...
+                    <Typography variant="h4" component="h1" className={classes.bigWhiteNumber}>
                       6
                     </Typography>
                     <Typography variant="body1" className={classes.thickBodyText}>
                       resource packs in use
                     </Typography>
                   </div>
                   <div className={classes.cbiItem}>
-                    <Typography variant="h4" className={classes.bigWhiteNumber}>
+                    <Typography variant="h4" component="h1" className={classes.bigWhiteNumber}>
```
Comment (autumnfound) [inline src/main/www/src/components/Portal/Dashboard/DashboardCommittersAndContributors.tsx:136]: h1 likely isn't the right choice here, as that serves as an effective new root heading, which these aren't

Comment (linkfang) [reply src/main/www/src/components/Portal/Dashboard/DashboardCommittersAndContributors.tsx]: No, definitely not.
I did a replace all to fix the header order issue and did not notice I messed up this part, lol.
Good catch, fixed!

**Review by autumnfound — APPROVED** [review]

**Pull request merged**